test/config/mysql/mysql.sql

```diff
@@ -0,0 +1,3 @@
+GRANT ALL ON *.* TO 'sequelize_test'@'%' with grant option;
```

Comment:
MySql is failing because the user `sequalize_test` doesn't have permissions to use the created database.

I noticed that MariaDB has a file like this to grant it privileges but I don't know how to do the same in mysql.

@papb any ideas?

Reply:
Hmm... Note that [this line in .travis.yml](https://github.com/sequelize/sequelize/blob/8b53f721a07e4ea5a42d74199b319f813f2c109c/.travis.yml#L31) tells MariaDB that it should use that file. I am not sure if something similar exists for MySQL, but if it does, you'll have to update the .travis.yml to do something similar.

Another options that comes to my mind is trying the following at the beginning of your test:

```js
sequelize.query(`GRANT ALL ON *.* TO 'sequelize_test'@'%' with grant option;`)
```

Reply:
I tried your second option but it just threw an error saying `sequalize_test` didn't have permission lol

Reply:
Hmm... Good luck :P